nemo/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "nemo"
+version = "0.1.1-dev"
+authors = [
+"Stefan Ellmauthaler <stefan.ellmauthaler@tu-dresden.de>",
+"Lukas Gerlach <lukas.gerlach@tu-dresden.de>",
+"Philipp Hanisch <philipp.hanisch1@tu-dresden.de>",
+"Alex Ivliev <alex.ivliev@tu-dresden.de>",
+"Markus Krötzsch <markus.kroetzsch@tu-dresden.de>",
+"Maximilian Marx <maximilian.marx@tu-dresden.de>"
+]	
+edition = "2021"
+
+[features]
+default = ["timing"]
+# Allows building for web assembly environments
+# Enables the "js" feature of the "getrandom" crate
+# This feature cannot be used together with the "timing" feature, because the "howlong" crate does not support web assembly environments
+js = ["getrandom/js"]
+# Enables time measurements using the "howlong" crate
+# If this feature is not enabled, all time measurements will display zero instead
+timing = ["dep:howlong"]
+no-prefixed-string-dictionary = []
```

Comment:
This should be propagated to `nemo-physical`, otherwise it will just silently do nothing.

Reply:
The actual dispatching happens in `nemo-physical/src/management/database.rs`, so this is where the flag ultimately needs to be set.